app/assets/javascripts/admin/components/theme-setting-relatives-selector.js.es6

```diff
@@ -0,0 +1,33 @@
+import Component from "@ember/component";
+import BufferedContent from "discourse/mixins/buffered-content";
+import SettingComponent from "admin/mixins/setting-component";
+import { ajax } from "discourse/lib/ajax";
+
+export default Component.extend(BufferedContent, SettingComponent, {
+  layoutName: "admin/templates/components/site-setting",
+
+  _save() {
+    return ajax(`/admin/themes/${this.model.id}`, {
```

Comment:
Is there a reason you couldn't use `this.model.save(...)` instead of a custom ajax call? If it uses the store the call should be what you've put here.